doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs1.rst

```diff
@@ -187,6 +187,15 @@ Security libraries
   * Added support for PSA crypto on the :ref:`nRF54L15 PDK <ug_nrf54l15_gs>`.
     Enabled the PSA crypto support with a CRACEN driver as the backend by setting the Kconfig option :kconfig:option:`CONFIG_NRF_SECURITY`.
 
+Scripts
```

Comment:
The content should be added to the "release-notes-changelog.rst" file. https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/releases_and_maturity/releases/release-notes-changelog.html